conversation_reconstruction/dataflow/write_page_states_to_spanner/spanner_write_test.py

```diff
@@ -27,18 +31,32 @@ def test_spanner_write(self):
         raise Exception(e)
     self.assertEqual(ret, 'Inserted data.')
     try:
-       ret = writer.insert_data('page_states', [('test_page_id', 'test_authors', 'test_conversation_id',
-                                  'test_deleted_comments', 'test_page_state', 124,
-                                  '2018-06-29T00:00:00Z')])
+      ret = writer.insert_data('page_states', {'page_id': 'test_page_id', 'authors': 'test_authors',
+                                               'conversation_id': 'test_conversation_id',
+                                               'deleted_comments': ['test_deleted_comment1', 'test_deleted_comment2'],
+                                               'page_state': 'test_page_state',
+                                               'rev_id': 124,
+                                               'timestamp': '2018-06-29T00:00:00Z'})
     except Exception as e:
       if 'StatusCode.ALREADY_EXISTS' in str(e):
         ret = 'Inserted data.'
         pass
       else:
         raise Exception(e)
-
     self.assertEqual(ret, 'Inserted data.')
-    pass
+    # Test Cloud Storage Input
+    cmd = os.system("python dataflow_main.py --input_storage=gs://wikidetox-viz-dataflow/WikiConv_v1/wikiconvs/test.json\
```

Comment:
I don't think you want to be testing this by os.system. This is all python code. You want to be writing tests in terms of small collections of local data in something like a directory called `testdata`.